<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_Menu</class>
 <widget class="QWidget" name="GUI_Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#opendocTab,QWidget#newdocTab{
	background-color: qlineargradient(spread:reflect, x1:0.5, y1:0, x2:0.5, y2:0.49, stop:0 rgba(207, 245, 236, 255), stop:0.17 rgba(215, 255, 245, 255), stop:1 rgba(230, 255, 249, 255));
}

QTabWidget::pane {
    border: 1px solid hsv(165, 55, 205);
}

QTabBar::tab {
    border: 2px solid hsv(165, 55, 205);
    /*min-width: 8ex;*/
    padding-left: 5px;
    padding-right: 5px;
    padding-top: 2px;
    padding-bottom: 2px;
}

QTabBar::tab:selected {
	background: rgba(207, 245, 236, 255);
    border-color: hsv(165, 55, 205);
	border-bottom: 0px;
/* make use of negative margins for overlapping tabs */
	/*margin-left: -4px;
    margin-right: -4px;*/
}

QTabBar::tab:!selected {
	background: hsv(165, 55, 205);
    margin-top: 2px; /* make non-selected tabs look smaller */
}

QTabBar::tab:!selected:hover {
	background: hsv(165, 55, 225);
    border-color: hsv(165, 55, 225);
}

QPushButton {
	min-height: 25px;
	min-width: 75px;
	padding: 5px;	
	padding-left: 10px;
	padding-right: 10px;
    background-color: hsv(165, 55, 255);
    border-width: 1px;
	border-style: outset;
	border-color: hsv(165, 55, 205);
}

QPushButton:hover:!pressed{
	border-color: hsv(165, 255, 255);
	border-top-color: hsv(165, 200, 255);	
	border-left-color: hsv(165, 200, 255);
}

QPushButton:pressed,QPushButton:checked {
    border-style: inset;
	background-color: hsv(165, 55, 220);
}

QLineEdit{
    border-width: 2px;
	border-style: inset;
	border-color: hsv(165, 55, 205);
}

QLineEdit:focus{
	border-color: hsv(165, 105, 205);
}

QComboBox{
	background-color: hsv(165, 55, 255);
    border-width: 1px;
	border-style: outset;
	border-color: hsv(165, 55, 205);
}

QComboBox QAbstractItemView {
    background-color: rgba(230, 255, 249, 255);
   /* border-width: 1px;
	border-style: outset;
	border-color: hsv(165, 55, 205);*/
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="userGroupBox">
     <property name="title">
      <string>User</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="iconLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/icon/Mew.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="nicknameLabel">
        <property name="cursor">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="text">
         <string>&lt;nickname&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>174</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="editPushButton">
        <property name="text">
         <string>Edit profile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="logoutPushButton">
        <property name="text">
         <string>Log out</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="documentGroupBox">
     <property name="title">
      <string>Documents</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="opendocTab">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <attribute name="title">
          <string>Accessible documents</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="rightMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
        <widget class="QWidget" name="newdocTab">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <attribute name="title">
          <string>New document</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3"/>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
